crates/runtime/src/lib.rs

```diff
@@ -466,6 +469,8 @@ impl Runtime {
 
         if cfg!(feature = "models") {
             futures.push(Box::pin(self.load_models()));
+        } else {
+            tracing::error!("Cannot load models without the 'models' feature enabled. {ENABLE_MODEL_SUPPORT_MESSAGE}");
```

Comment:
Similarly, this will always log. We should only log if the user attempts to use a functionality that needs the `models` feature.